topics/dwd/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: deutscherwetterdienst, deutscher-wetterdienst
+display_name: Deutscher Wetterdienst
+github_url: https://github.com/DeutscherWetterdienst
+short_description: The DWD is the German Meteorological Service.
+topic: dwd
+wikipedia_url: https://en.wikipedia.org/wiki/Deutscher_Wetterdienst
+url: https://www.dwd.de/DE/Home/home_node.html
+related: open-data, climate-change
+---
+The Deutscher Wetterdienst or DWD for short, is the German Meteorological Service, based in Offenbach am Main, Germany. The DWD monitors, similar to NOAA in the USA, weather and meteorological conditions over Germany and provides weather services for the general public and for nautical, aviational or agricultural purposes.
+
+Since 2017 the DWD provides it's data partially to the public as [open data](https://opendata.dwd.de).
```

Comment:
Tiny grammar nit: this should be a possessive "its" with no apostrophe 👀 

```suggestion
Since 2017 the DWD provides its data partially to the public as [open data](https://opendata.dwd.de).
```